src/stars.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Why do you write styles for the body inside the stars file? Move it to the main CSS file